scripts/la_selva_trees.py

```diff
@@ -8,7 +8,7 @@
 class main(Script):
     def __init__(self, **kwargs):
         Script.__init__(self, **kwargs)
-        self.tables={'trees': Table('trees', cleanup=Cleanup(correct_invalid_value, missingValues=[-999]))}
+        self.cleanup_func_table = Cleanup(correct_invalid_value, missing_values=[-999.9])
```

Comment:
Was there a reason to change this from `-999` to `-999.9`? Looks like this broke the current null value handling (at least on 2.0.0).